lib/grammars.coffee

```diff
@@ -868,3 +868,12 @@ module.exports =
         uri = 'file://' + context.filepath
         shell.openExternal(uri)
         ['HTML file opened at:', uri]
+  
+  MQL5:
+    if GrammarUtils.OperatingSystem.isDarwin()
```

Comment:
Also, this conditional needs to be applied to a value response, not as part of this structure. For now, since this seems only Windows based, I would take this conditional check out.